### PR TITLE
Rework `_cap_csr`

### PR DESCRIPTION
**Pull request by dshil**

### CODE REVIEW

File: gits/terminal.py [hunk 1]
```diff
@@ -297,6 +297,8 @@ def _cap_csr(self, top, bottom):
         """
         self._top_most = min(self._rows - 1, top - 1)
         self._bottom_most = min(self._rows - 1, bottom - 1)
+
+        # `_bottom_most` must be greater or equal `_top_most`.
```
Comment (eugulixes) [inline gits/terminal.py]: ...must be greater _than_ or equal _to_...

File: gits/terminal.py [hunk 2]
```diff
@@ -297,6 +297,8 @@ def _cap_csr(self, top, bottom):
         """
         self._top_most = min(self._rows - 1, top - 1)
```
Comment (eugulixes) [inline gits/terminal.py]: I think we have to substitute `self._rows - 1` for `self._bottom_most`.

File: gits/terminal.py (hunk 1 above)
Comment (dshil) [inline gits/terminal.py]: fxd, ping @eugulixes

**Comment by dshil** [comment]
Comments were taken into account, ping @eugulixes .

**Review by eugulixes — APPROVED** [review]

**Pull request merged**